lib/_http_server.js

```diff
@@ -904,6 +907,17 @@ function parserOnIncoming(server, socket, state, req, keepAlive) {
          resOnFinish.bind(undefined,
                           req, res, socket, state, server));
 
+  if (typeof server.maxRequestsPerSocket === 'number'
+    && (req.httpVersionMajor === 1 && req.httpVersionMinor === 1)) {
+
+    if (server.maxRequestsPerSocket < ++state.requestsCount) {
+      res.writeHead(503);
+      res.end();
+    }
+
+    res.shouldKeepAlive = server.maxRequestsPerSocket > state.requestsCount
```

Comment:
```suggestion
    res.shouldKeepAlive = server.maxRequestsPerSocket >= state.requestsCount
```

I think this should go to false if they are the same.

Reply:
In the current code the `shouldKeepAlive ` will be `false` if they are equal
the `shouldKeepAlive ` is set to true only if `requestsCount` is small enough

```
server.maxRequestsPerSocket > state.requestsCount
```